### ISSUE TITLE
Can you Support Yarn >= 1.3.1 ?

### ISSUE THREAD

**Issue by hiyamamo** [masawada/yarn-outdated-formatter#23]
```
$ yarn outdated --json | $(yarn bin)/format-yarn-outdated
undefined:1



SyntaxError: Unexpected end of JSON input
    at Object.parse (native)
    at Socket.process.stdin.on (/Users/yamamoto/sandbox/delight/node_modules/yarn-outdated-formatter/bin/format-yarn-outdated.js:48:27)
    at emitNone (events.js:91:20)
    at Socket.emit (events.js:185:7)
    at endReadableNT (_stream_readable.js:974:12)
    at _combinedTickCallback (internal/process/next_tick.js:80:11)
    at process._tickCallback (internal/process/next_tick.js:104:9)
```

```
$ yarn --version
1.3.1
```

**Comment by masawada** [comment]
Thank you for your report! and sorry for my late response:bow:
This problem was fixed in #24 by @tsuyoshiwada (thanks!). and changes will be published soon!
Please wait a little more:pray:

**Comment by masawada** [comment]
`yarn-outdated-formatter@2.0.0` has been released. Please check it out!
